SCSS/style.scss

```diff
@@ -0,0 +1,992 @@
+                button::before{
+                    color: white;
+                }
+            }
+            .slick-next{
+                right:0px; 
+            }
+       }
+       .slick-arrow{
+        margin-top: 0vh;
+            &.slick-prev{
+                &::before {
+                    font-family: "Font Awesome 6 Free" !important ;
+                    content: '\f053' ;
+                    color: white ;
+                    font-weight: 900;
+                    font-size: 50px;
+                  }
+                @media only screen and (max-width:600px) {
+                    &::before {
+                    font-family: "Font Awesome 6 Free" !important ;
+                    content: '' ;
+                  
+                  }
+                }
+            }
+            
+            
+        &.slick-next::before {
+           
+            content: '\f054';
+            color: white;
+            font-weight: 900;
+            font-size: 50px;
+            font-family: "Font Awesome 6 Free";
+        
+     
+        }
+        @media only screen and (max-width:600px) {
+            &.slick-next::before  {
+            font-family: "Font Awesome 6 Free" !important ;
+            content: '' ;
+          
+          }
+    }
+        }
+        .inner-div{
+            width: 60%;
+            margin: 20vh auto;            
+            padding-top: 50px;
+            text-align: center;
+            color: white;
+            align-content: center;
+            h3{
+                font-size: 2rem;
+                font-weight: 300;
+            }
+            h1{
+                font-size: 3rem;
+            }
+            p{
+                width: 60%;
+                margin: 10px auto 50px auto;
+
+            }
+            @media only screen and (max-width:576px) {
+                p  {
+                width: 100%;
+              
+              }
+        }
+            .banner-btn{
+                color: white;
+                background-color: brown;
+            }
+            
+        }
+        @media only screen and (max-width:758px) {
+            .inner-div  {
+            width: 80%;
+          
+          }
+    }
+        
+    }
+
+    // banner slider ends 
+    
+   
+}
+@media only screen and (max-width:768px){
+    .banner{
+        height: 70vh;
+        .banner-div{
+            .slick-slider{
+                .slick-dots{
+                    bottom: -80px;
+                    button::before{
+                        color: white;
+                    }
+                }
+                
+           }
+            .inner-div{
+                width: 80%;
+                margin: 10vh auto;            
+                text-align: center;
+                color: white;
+                align-content: center;
+                p{
+                    width: 80%;
+                }
+            }
+        }
+        
+    }
+}
+@media only screen and (max-width:576px){
```

Comment:
this is not the perfect way to use media query please fix this

Reply:
done